### PR TITLE
Adding 2 new ElecMu paths to be monitored in topHLTOfflineDQM package

### PR DESCRIPTION
**Pull request by soureek**
In response to JIRA ticket [1] , adding the following two ElecMu paths to be monitored in the topHLTOfflineDQM package. 
HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ_v*
HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v*

As explained in [1], these two paths will act as shadow to the following paths. HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ_v*
HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v*

The package has been tested to be OK by running the matrix corresponding to TTbar_13 (25ns) workflow (250202.0).

[1] https://its.cern.ch/jira/browse/CMSHLT-961

### CODE REVIEW

File: DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py [hunk 1]
```diff
@@ -335,7 +339,9 @@
 		pathsELECMU = cms.vstring([ 'HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v',
                                     'HLT_Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_v',
                                     'HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_v', 
-                                    'HLT_Mu8_TrkIsoVVL_Ele17_CaloIdL_TrackIdL_IsoVL_v']),
+                                    'HLT_Mu8_TrkIsoVVL_Ele17_CaloIdL_TrackIdL_IsoVL_v',
+									'HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ_v',
```
Comment (dmitrijus) [inline DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py]: Please fix your text editor not emit tabs when the entire file uses spaces.

File: DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py [hunk 2]
```diff
@@ -501,12 +471,7 @@
     trigger = cms.PSet(
         src    = cms.InputTag("TriggerResults","","HLT"),
 ### Updating to HLT paths to be monitored by TOP PAG in 2016 
-		select = cms.vstring(['HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v',                                                                                            
-                              'HLT_Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_v',
-                              'HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_v',
-                              'HLT_Mu8_TrkIsoVVL_Ele17_CaloIdL_TrackIdL_IsoVL_v',
-							  'HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ_v',
-							  'HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v'])
+		select = cms.vstring(['HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v','HLT_Mu23_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_v','HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_v','HLT_Mu8_TrkIsoVVL_Ele17_CaloIdL_TrackIdL_IsoVL_v','HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_DZ_v','HLT_Mu12_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v'])
```
Comment (dmitrijus) [inline DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py]: This is still 2 tabs! I usually skip past such errors in cpp, but in python space errors are somewhat critical.

Comment (Soureek89) [inline DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py]: The two tabs that you are mentioning in your comment have been removed indicated by the minus sign on the left in comparison that you have sent. I have put all the path names (strings) one after another separated by comma in a single line.

Comment (dmitrijus) [inline DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py]: Make sure you have zero tabs in the entire file!

You fixed the ones I have mentioned before, but the file is still full of them. For example, the line I've commented, has two tabs just before "select = ".

**Comment by Soureek89** [comment]
@dmitrijus : I have removed all tab spaces from DQMOffline/Trigger/python/topDiLeptonHLTEventDQM_cfi.py

Please check.

**Comment by dmitrijus** [comment]
+1

**Comment by Soureek89** [comment]
Previous test failed due to DAS error as shown in Matrix Tests Outputs in [1]. Upon further investigation of the corresponding log file[2] I figured that there was a "DAS Error" for workflow **140.53_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI**
The full status for this particular workflow shows the following.
Step0-DAS_ERROR Step1-NOTRUN Step2-NOTRUN  - time date Wed Aug  3 15:12:28 2016-date Wed Aug  3 15:11:54 2016; exit: 1 0 0 

[1] https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15312/14341/summary.html 
[2] https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15312/14341/runTheMatrix-results/matrixTests.log

**Comment by perrotta** [comment]
please test

**Pull request closed without merging**